pages.de/common/cat.md

```diff
@@ -9,16 +9,16 @@
 
 - Verkette mehrere Dateien und speichere das Ergebnis in einer neuen Datei:
 
-`cat {{pfad/zu/datei1}} {{pfad/zu/datei2}} > {{pfad/zu/ziel_datei}}`
+`cat {{pfad/zu/datei1 pfad/zu/datei2 ...}} > {{pfad/zu/ziel_datei}}`
 
 - Verkette mehrere Dateien und hänge das Ergebnis an eine Datei an:
 
-`cat {{pfad/zu/datei1}} {{pfad/zu/datei2}} >> {{pfad/zu/ziel_datei}}`
+`cat {{pfad/zu/datei1 pfad/zu/datei2 ...}} >> {{pfad/zu/ziel_datei}}`
 
-- Nummeriere alle ausgegebenen Zeilen:
+- Kopieren den Inhalt einer Datei ohne Pufferung in eine Ausgabedatei:
```

Comment:
```suggestion
- Kopiere den Inhalt einer Datei in eine Ausgabedatei ohne zu puffern:
```
Please do not use an online translator, I'm happy to help out with translations if pinged.